JDK8 & JDK11  and JDk14 build failing due to Installer issue : Execution error: null

jdk8 & jdk11 jdk14 nightly builds failing with the same error

https://ci.adoptopenjdk.net/view/Failing%20Builds/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-linux-aarch64-hotspot/568/console

Installer issue 
`Execution error: null`

```
> Task :deb:uploadDebianPackage FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':deb:uploadDebianPackage' (type 'UploadDebianPackage').
> No value has been specified for property 'repository'.
```

--- a/linux/build.gradle
+++ b/linux/build.gradle
@@ -42,11 +42,11 @@ ext {
     ]
     artifactory = [
         url       : "https://adoptopenjdk.jfrog.io/adoptopenjdk/",
-        user      : hasProperty("ARTIFACTORY_USER") ? ARTIFACTORY_USER : null,
-        password  : hasProperty("ARTIFACTORY_PASSWORD") ? ARTIFACTORY_PASSWORD : null,
+        user      : project.hasProperty("ARTIFACTORY_USER") ? ARTIFACTORY_USER : null,
+        password  : project.hasProperty("ARTIFACTORY_PASSWORD") ? ARTIFACTORY_PASSWORD : null,
         repository: [
-            deb: hasProperty("ARTIFACTORY_REPOSITORY_DEB") ? ARTIFACTORY_REPOSITORY_DEB : null,
-            rpm: hasProperty("ARTIFACTORY_REPOSITORY_RPM") ? ARTIFACTORY_REPOSITORY_RPM : null,
+            deb: project.hasProperty("ARTIFACTORY_REPOSITORY_DEB") ? ARTIFACTORY_REPOSITORY_DEB : null,
+            rpm: project.hasProperty("ARTIFACTORY_REPOSITORY_RPM") ? ARTIFACTORY_REPOSITORY_RPM : null,
         ],
     ]
     jdkDistributionDir = JDK_DISTRIBUTION_DIR

